"Fix build errors due new gcc 10 default for -fno-common"
https://github.com/amadvance/advancemame/commit/f8688cae05799a30cd5337e626a07fa9f004c0a3

Index: src/drivers/cavepgm.c
--- src/drivers/cavepgm.c.orig
+++ src/drivers/cavepgm.c
@@ -287,7 +287,7 @@ Notes:
 #include "timer.h"
 
 
-UINT16 *pgm_mainram, *pgm_bg_videoram, *pgm_tx_videoram, *pgm_videoregs, *pgm_rowscrollram;
+extern UINT16 *pgm_mainram, *pgm_bg_videoram, *pgm_tx_videoram, *pgm_videoregs, *pgm_rowscrollram;
 static UINT8 *z80_mainram;
 static UINT32 *arm7_shareram;
 static UINT32 arm7_latch;
@@ -852,8 +852,8 @@ static void expand_32x32x5bpp(void)
 /* This function expands the sprite colour data (in the A Roms) from 3 pixels
    in each word to a byte per pixel making it easier to use */
 
-UINT8 *pgm_sprite_a_region;
-size_t	pgm_sprite_a_region_allocate;
+extern UINT8 *pgm_sprite_a_region;
+extern size_t	pgm_sprite_a_region_allocate;
 
 static void expand_colourdata(void)
 {
